services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,32 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+   xQueueSend(thermalMgrQueueHandle, event, 0); // why u throwing error
```

Comment:
- [ ] check to make sure that thermal mgr queue isn't null before use, returning invalid state
- [ ] check to make sure that the event isn't null
- [ ] in general, you should be checking the return value of non void functions for errors and handling them accordingly